### DISCUSSION TITLE
Extends: merge public dir

### DISCUSSION THREAD

**Discussion by Phivan** [nuxt/nuxt#18472] [Questions]
Hello everyone.

Is it possible to merge public dir when using extends key ? Just like components and composables.

I have public assets (images) that will be used in the futur by others applications and some very specific.

When using `npm run build` with a public image referenced in base application I got this error: 

`Rollup failed to resolve import "/images/my/path/img.png" from "../relative/path/to/base/Component.vue".`

I guess that if my app's base public assets is merged in my children public asset, it will work because I tried:
    ```
dir:{
      public:'/my/path/to/base/public/'
    }```
and build runs without error. Off course the result is not OK because specific assets are missing in .ouput.

Thank you all for you help. 

(And sorry for my terrible English :) )

**Comment by TheAlexLichter** [comment]
Is [this discussion](https://github.com/nuxt/nuxt/discussions/16215) possibly about the same issue?

**Reply by Phivan** [reply]
First I edited my question, a copy/past error in configuration that "work". Sorry for that

Your link seems to be related, but I'm not sure to understand the answser. How and where I am supposed to implement copyPublicAssets ?
